### PR TITLE
New Xmonad Cheat Sheet

### PR DESCRIPTION
**Pull request by jarroddunne**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [x] New Instant Answer
    - [ ] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [ ] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

Adds a cheatsheet for the xmonad window manager default keybindings.

###### Which issues (if any) does this fix?

Fixes #NNNN - how specifically does it fix the issue?

###### People to notify (@mention interested parties)

---

Instant Answer Page: https://duck.co/ia/view/xmonad_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @mention

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=xmonad cheatsheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/xmonad_cheat_sheet_mobile.png?PUpV59uz6b)

File: share/goodie/cheat_sheets/json/xmonad.json [hunk 1]
```diff
@@ -0,0 +1,120 @@
+{
+    "id": "xmonad_cheat_sheet",
+    "name": "Xmonad",
+    "description": "Default keybindings for Xmonad window manager",
+    "metadata": {
+        "sourceName": "xmonad manpage from xmonad.org",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/xmonad.json]: "xmonad" would be enough here

File: share/goodie/cheat_sheets/json/xmonad.json [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+        "sourceUrl" : "http://xmonad.org/manpage.html"
+    },
+
+    "aliases": [ "xmonad"
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/xmonad.json]: This alias is same as file name. so need of this one. If there are other aliases, lets add them.

**Comment by jarroddunne** [comment]
@jophab Which line were you referring to with "'xmonad' would be enough here'". Thanks!

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=xmonad cheatsheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/xmonad_cheat_sheet_mobile.png?QeR4YesAHJ)

**Comment by jophab** [comment]
I mean the source name to be changed to "xmonad"

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=xmonad cheatsheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/xmonad_cheat_sheet_mobile.png?VQn5Qm5cEo)

**Comment by tagawa** [comment]
@jarroddunne Thank you very much for creating this and implementing @jophab's comments. It's looking good [on the beta server](https://beta.duckduckgo.com/?q=xmonad%20cheatsheet). Just one thing - I wonder if it would be better to split the shortcut list into two sections that would require less scrolling for users. For example, a section for "Window Management" (resizing, moving focus, etc.), and one for "General" or "Other" (restarting, launching, etc.). What do you think?

**Comment by jarroddunne** [comment]
@tagawa Thanks for the suggestion. I went ahead and separated the cheatsheet into three sections, Window Management, System Options, and Workspace/Screen Management.

The screenshot from @daxtheduck shows "More at %s" instead of the correct url for the original source. I checked it on the duckpan server from codio, and it showed the source url, is this just a problem with the deployment screenshotting system?

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=xmonad cheatsheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/xmonad_cheat_sheet_mobile.png?TX7srjxRf5)

**Comment by tagawa** [comment]
@jarroddunne Cool - this looks good to me! Yes, the `%s` is a screenshot-only bug. It looks fine on the beta server where it's now available for public testing: https://beta.duckduckgo.com/?q=xmonad+cheatsheet&iax=1

We'll keep this open for review comments for a few days and then merge if all is well.

**Comment by tagawa** [comment]
Looking good here and no further feedback so let's merge! Thank you again for creating this @jarroddunne and to @jophab for the review comments. 🍰

**Comment by daxtheduck** [comment]
@jarroddunne this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Pull request merged**